tooling/metamodel-generator/src/main/java/org/hibernate/processor/HibernateProcessor.java

```diff
@@ -251,6 +252,12 @@ private boolean handleSettings(ProcessingEnvironment environment) {
 		return parseBoolean( options.get( FULLY_ANNOTATION_CONFIGURED_OPTION ) );
 	}
 
+	private static boolean packagePresent(@Nullable PackageElement pack) {
+		return pack != null
+			//HHH-18019 ejc always returns a non-null PackageElement
+			&& !pack.getEnclosedElements().isEmpty();
```

Comment:
Ha, nice - was just about to propose the same workaround (coming from https://github.com/quarkusio/quarkus/issues/40809).

@gavinking @yrodiere Are you going to report a bug for ECJ by any chance?

Reply:
@famod I wasn't intending to as I suspect there's a reason for this (we'd need to check past bug reports) and I'm traveling this week anyway, but if you can that'd be great!

Reply:
@famod The Javadoc for that method isn't well-written, but it does _imply_ that the method is supposed to search for "non-empty packages".